src/api/apiHelpers.js

```diff
@@ -15,7 +15,7 @@ export function createVolData(
     name,
     email,
     phoneNumber,
-    vehicleAccess,
+    vehicleAccess: vehicleAccess === 1,
```

Comment:
added this so that inside the csv, it would show up as TRUE instead of 1. 

<img width="81" alt="image" src="https://user-images.githubusercontent.com/8000986/141385876-e202f77c-813c-4f05-b661-106e94c0d7d2.png">